Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -71,7 +71,11 @@ script:
             throw 'Error parsing reply - ' + result.Body + ' - ' + ex;
         }
         if (obj.errors && obj.errors.length > 0) {
-            throw JSON.stringify(obj.errors);
+            if (dq(obj, 'errors.[0].code') === 404 && dq(obj, 'errors.[0].message').indexOf('Resource Not Found') > -1) {
```

Comment:
I think this will fail in line 62 as the code will be 404 with this body... would check and handle differently in case of 404 is status code